wondrous-app/components/Common/Share/index.tsx

```diff
@@ -7,16 +7,18 @@ import { StyledShare } from './styles';
 interface IShareProps {
   url: string;
   className?: string;
+  isBounty?: boolean;
 }
 
 export const Share = (props: IShareProps) => {
-  const { url, className } = props;
+  const { url, className, isBounty = false } = props;
   const { setSnackbarAlertOpen, setSnackbarAlertMessage } = useContext(SnackbarAlertContext);
+  const snackbarAlertMessage = isBounty ? 'Bounty link copied' : 'Task link copied';
```

Comment:
can we have a more general solution to this? currently we have 4 types: Tasks, Milestones, Bounties, Proposals

it would be great if we could send the type and in here check for each specific type

e.g instead of isBounty pass the task type and then have a object holding different labels and then we can do ${label / taskType} link copied

because we might run in the same issue with milestones and proposals

Reply:
makes sense, let me try and rework the fix

Reply:
have made the changes, could you please have a look @Lamperoyge ?

Reply:
nice, ty, approved

Reply:
thank you :)